itou/utils/perms/prescriber.py

```diff
@@ -1,15 +1,11 @@
 from django.db.models import Q
-from django.shortcuts import get_object_or_404
-
-from itou.prescribers.models import PrescriberOrganization
-from itou.utils import constants as global_constants
+from django.http import Http404
 
 
 def get_current_org_or_404(request):
-    pk = request.session.get(global_constants.ITOU_SESSION_CURRENT_ORGANIZATION_KEY)
-    queryset = PrescriberOrganization.objects.member_required(request.user)
-    organization = get_object_or_404(queryset, pk=pk)
-    return organization
+    if request.current_organization and request.user.is_prescriber:  # Set by middleware for prescriber users
```

Comment:
nit: Peut-être changer l’order des conditions autour du `and` pour correspondre à `get_current_institution_or_404` et `get_current_siae_or_404` ?

Ça permet aussi de vérifier qu’on manipule bien un utilisateur du bon type.

Reply:
Bien vu, c'était effectivement mon souhait mais j'ai oublié pour les prescripteurs :+1: